Assets/Scripts/Characters/Modifier.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+/*
+ * A modifier encapsulates the idea of a buff or debuff. It can be multiplicative or additive (or both).
```

Comment:
The both was just a side effect of the math. There is no _reason_ that I can think of that a modifier should do both. Just, as it's written right now, a single modifier could have both a multiplicative component and an additive component and nothing weird would happen.